data/names/en.json

```diff
@@ -2515,6 +2517,7 @@
 {"usage": "city", "name": "New Sharon"},
 {"usage": "city", "name": "New Shoreham"},
 {"usage": "city", "name": "New Sweden"},
+{"usage": "city", "name": "New Vegas"},
```

Comment:
These are generally based on real city names - is there a New Vegas somewhere?

Reply:
> These are generally based on real city names - is there a New Vegas somewhere?

There most likely is not. It's a reference to [A classic post apocalyptic video game of the same name](https://en.wikipedia.org/wiki/Fallout:_New_Vegas).

Reply:
Let's keep the place names to real ones, it's best we don't go down the road of allowing references in this particular context.

Reply:
> Let's keep the place names to real ones, it's best we don't go down the road of allowing references in this particular context.

I gotcha, I'll get rid of that one.

All of the other places are real, so they should be good then.